docs/quickstart/basics.rst

```diff
@@ -19,7 +19,8 @@ Let's take a look at a very basic API using Flask-Rebar:
 
 
    class TodoSchema(ResponseSchema):
-       id = fields.Integer()
+       id = fields.Integer(dump_only=True)
+       description = fields.String(required=True)
```

Comment:
I personally like having different schema for create and get, but this pattern is also nice so I am not sure what we should recommend to beginners.

Reply:
Check out the latest revision (which actually includes a little more explanation in the docs) and let me know what you think.

When the data for the same type of object is for some reason very different in request bodies compared to response bodies, it makes sense to use different schemas. But in the simpler case (which is maybe more common?) where the only difference is that some fields should only be read-only by users, since they're managed entirely by the database or application, then I think using a single Schema with those fields marked as `dump_only` is a perfect fit. Common examples in my experience are autogenerated `id`s and automatically maintained `updated_at` and `created_at` timestamps, which can all be maintained by a SQL database automatically.

FWIW, this is the pattern I use in the class I've been teaching to beginners and it's been working really well. (For the past year I've been teaching a monthly class at Two Sigma on how to build REST APIs with Flask-Rebar, and by the time they leave the class, students have built a full working CRUD app to manage (you guessed it) their Todo list. 🤣)

Reply:
Definitely much better with the new comments, I am not sure about the form (explanation before/after code vs inline comments) but that can be improved later.